### DISCUSSION TITLE
unable to login with correct username/password

### DISCUSSION THREAD

**Discussion by reedone816** [xterm-inator/MikroGuard#54] [Q&A]
after succesfully running the mikroguard i access the website [ip address]:8000
with successful login, no redirect, still showing the login page.
if the username/password wrong it gives warning.
what should i do?

i found this in the running log:
MikroGuard  | [2024-11-26 02:54:15] production.ERROR: Session store not set on request. {"userId":1,"exception":"[object] (RuntimeException(code: 0): Session store not set on request. at /opt/app/vendor/laravel/framework/src/Illuminate/Http/Request.php:564)

**Comment by xterm-inator** [comment]
hey, 

normally this is when the api and browser are using different addresses, have a look at [troubleshooting](https://github.com/xterm-inator/MikroGuard?tab=readme-ov-file#troubleshooting), and see if that helps
